meta-luneui/recipes-upstreamable/uriparser/uriparser_0.9.3.bb

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2012-2014 LG Electronics, Inc.
+
```

Comment:
uriparser is already in meta-oe (the 0.8.4 version) we should add this upgrade there and drop it from meta-luneui.

I'll upstream the upgrade.